sdgym/result_explorer/result_explorer.py

```diff
@@ -14,20 +18,63 @@ def _validate_local_path(path):
         raise ValueError(f"The provided path '{path}' is not a valid local directory.")
 
 
+_BASELINE_BY_MODALITY = {
+    'single_table': SYNTHESIZER_BASELINE,
+    'multi_table': 'MultiTableUniformSynthesizer',
```

Comment:
@fealho can you ask in the Team-Engineering channel which synthesizer we should use as the baseline for multi-table here?

Just to make sure everything is as expected. With this implementation it will be easy to update it in the future 👍